es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/ElasticSearchDAOV6.java

```diff
@@ -310,20 +298,20 @@ private void createIndex(String indexName) {
                         .create(createIndexRequest)
                         .actionGet();
             } catch (ResourceAlreadyExistsException done) {
-                logger.error("Failed to update log index name: {}", indexName, done);
+                LOGGER.error("Failed to update log index name: {}", indexName, done);
             }
         }
     }
 
     private void addTypeMapping(String indexName, String type, String sourcePath) {
         GetMappingsResponse getMappingsResponse = elasticSearchClient.admin().indices().prepareGetMappings(indexName).addTypes(type).execute().actionGet();
         if (getMappingsResponse.mappings().isEmpty()) {
-            logger.info("Adding the {} type mappings", indexName);
+            LOGGER.info("Adding the {} type mappings", indexName);
             try {
                 String source = loadTypeMappingSource(sourcePath);
                 elasticSearchClient.admin().indices().preparePutMapping(indexName).setType(type).setSource(source, XContentType.JSON).execute().actionGet();
             } catch (Exception e) {
-                logger.error("Failed to init index " + indexName + " mappings", e);
+                LOGGER.error("Failed to init index " + indexName + " mappings", e);
```

Comment:
When you get this error, doesn't the whole system end up in a bad state? Here you're just logging and continuing on, but something really bad happened. Does it make sense to let the process go on?